test/e2e/utils.go

```diff
@@ -315,24 +312,28 @@ func currentState(namespace string, node string, currentStateYaml *nmstatev1alph
 	}, ReadTimeout, ReadInterval)
 }
 
-func checkCondition(node string, conditionType nmstatev1alpha1.ConditionType) AsyncAssertion {
-	key := types.NamespacedName{Name: node}
+func checkEnacmentConditionEventually(node string, conditionType nmstatev1alpha1.ConditionType) AsyncAssertion {
+	key := types.NamespacedName{Name: DefaultPolicy}
 	return Eventually(func() corev1.ConditionStatus {
-		state := nodeNetworkState(key)
-		condition := state.Status.Conditions.Find(conditionType)
+		policy := nodeNetworkConfigurationPolicy(key)
+		condition := policy.Status.Enactments.FindCondition(node, conditionType)
 		if condition == nil {
 			return corev1.ConditionUnknown
 		}
 		return condition.Status
 	}, ReadTimeout, ReadInterval)
```

Comment:
In function `checkEnacmentConditionConsistently()` has time outs explicitly set. In this function it is set by variables. I would like a consistent style, please.

Reply:
Thanks, let's define per nature the timeout and internval.